NBXplorer/TrackedTransaction.cs

```diff
@@ -101,6 +101,8 @@ internal void KnownKeyPathMappingUpdated()
 			for (int i = 0; i < Transaction.Outputs.Count; i++)
 			{
 				var output = Transaction.Outputs[i];
+				if(output.Value == null )
+					continue;
```

Comment:
I think you can remove this.

Reply:
This is in case the matched output was not blinded. Basically, if you have another wallet generating keys on the same paths with the same xpub, and uses a different blinding key, this will save invalid(impartial) data

Reply:
it does not matter, we should still index that. That people can't read the value is their problem.

Reply:
done